projects/pgai/db/sql/idempotent/012-vectorizer-api.sql

```diff
@@ -603,8 +574,8 @@ create or replace view ai.vectorizer_status as
 select
   v.id
 , pg_catalog.format('%I.%I', v.source_schema, v.source_table) as source_table
-, pg_catalog.format('%I.%I', v.target_schema, v.target_table) as target_table
-, pg_catalog.format('%I.%I', v.view_schema, v.view_name) as "view"
+, pg_catalog.format('%I.%I', v.config operator(pg_catalog.->) 'destination' operator(pg_catalog.->>) 'target_schema', v.config operator(pg_catalog.->) 'destination' operator(pg_catalog.->>) 'target_table') as target_table
```

Comment:
should the target_table or "view" be the source table in the dest_column case? should we add an "embedding_column" info column to the view?

Reply:
I added the embedding_column info column, I'm not so sure about putting the source table also into the target table column, that seems a bit unintuitive to me?